includes/ajax-functions.php

```diff
@@ -724,9 +727,28 @@ function edd_ajax_download_search() {
 	// Done!
 	edd_die();
 }
-add_action( 'wp_ajax_edd_download_search',        'edd_ajax_download_search' );
+add_action( 'wp_ajax_edd_download_search', 'edd_ajax_download_search' );
 add_action( 'wp_ajax_nopriv_edd_download_search', 'edd_ajax_download_search' );
 
+/**
+ * Filters the WHERE SQL query for the edd_download_search.
+ * This searches the download titles only, not the excerpt/content.
+ *
+ * @since 3.1.0.2
+ * @param string $where
+ * @param WP_Query $wp_query
+ * @return string
+ */
+function edd_ajax_filter_download_where( $where, $wp_query ) {
+	$search = $wp_query->get( 'edd_search' );
```

Comment:
We should be use `esc_like` to do proper sanitization of the search query before using it.
https://developer.wordpress.org/reference/classes/wpdb/esc_like/